web-api/src/personalization/routes.rs

```diff
@@ -707,6 +744,17 @@ struct SemanticSearchRequest {
     is_deprecated: bool,
 }
 
+#[allow(clippy::too_many_arguments)]
+#[allow(clippy::struct_excessive_bools)]
```

Comment:
```suggestion
```

Both of this seem to not necessary (when I removed them on a local checkout, both with the tooling we use and nightly clippy).

Reply:
done